srcpkgs/simavr/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'simavr'
+pkgname=simavr
+version=1.6
+revision=1
+build_style=gnu-makefile
+make_build_args="RELEASE=1"
+make_build_target="build-simavr build-parts"
+make_use_env=1
+hostmakedepends="pkg-config"
+makedepends="elfutils-devel libfreeglut-devel"
+short_desc="A lean, mean and hackable AVR simulator"
+maintainer="Martijn van Buul <martijn.van.buul@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/buserror/${pkgname}"
+distfiles="https://github.com/buserror/${pkgname}/archive/v${version}.tar.gz"
+checksum=a55ad04d055eef5656c49f78bc089968b059c6efb6a831618b8d7e67a840936d
+
+CFLAGS+="-fPIC"
```

Comment:
`CFLAGS+="-fPIC"`

why?

Reply:
Well, two reasons:

1) Because upstream says so in the documentation
2) Because it won't build otherwise, as the shared library won't link.

The alternative would be to fix the build process, which is a bit more work - and would only be worthwhile if upstream would be willing to fix it. Which, given the lack of attention this has been getting over the last few years, is not very likely :-/

Reply:
It won't build otherwise? Really?
You take me for an idiot.
When I asked "why ?" it was because I was able to build this package without this useless flag.

Reply:
I deeply resent the way you interact with people. Your original response offended me greatly.

Do whatever you want. I'm abandoning this, and do not want to communicate with you any more.